app/controllers/api/v1/change_requests_controller.rb

```diff
@@ -1,7 +1,11 @@
 class Api::V1::ChangeRequestsController < Api::V1::ApplicationController
   before_action :check_token_and_set_application, only: %i[index], if: :json_request?
 
-  def index; end
+  def index
+    @documents_for_change_request = @planning_application.document_change_requests.map do |doc|
```

Comment:
@documents_for_change_request doesn't seem to be being used anywhere?

But also another way of doing this would be:

`@planning_application.document_change_requests.map(&:document)`